include/envoy/stats/allocator.h

```diff
@@ -49,6 +49,15 @@ class Allocator {
                                    const StatNameTagVector& stat_name_tags,
                                    Gauge::ImportMode import_mode) PURE;
 
+  /**
+   * @param name the full name of the stat.
+   * @param tag_extracted_name the name of the stat with tag-values stripped out.
+   * @param tags the tag values.
+   * @return TextReadoutSharedPtr a text readout, or nullptr if allocation failed, in which case
+   *     tag_extracted_name and tags are not moved.
```

Comment:
I don't think allocation can fail anymore (this is a holdover from the shared mem days), right? Can you fix this above also?

Reply:
We crash on OOM so I think these comments can just be deleted. Thank you!

/wait

Reply:
Thanks for the confirmation! Done.